Cities.txt

```diff
@@ -3,3 +3,5 @@ New Jersey
 Washington DC
 Miami
```

Comment:
What about adding LA instead of Centreville?

Reply:
sounds good